client/lib/posts/post-list-store.js

```diff
@@ -103,14 +103,16 @@ export default function( id ) {
 	}
 
 	/**
-	 * Sort the active list
-	 **/
-	function sort() {
+	 * sort post-list according to current orderBy parameter
+	 * @param  {array} list - list of posts to sort
+	 * @return {array}      sorted array (original will not be mutated)
+	 */
+	function sort( list ) {
 		var key = _activeList.query.orderBy;
 
-		_activeList.postIds.sort( function( a, b ) {
-			var postA = PostsStore.get( a ),
-				postB = PostsStore.get( b ),
+		return list.slice( 0 ).sort( function( a, b ) {
+			var postA = PostsStore.get( a ) || a,
```

Comment:
No, it's the raw Post object. I added the `|| a` because I wanted to sort the raw Post list from the server before I compare it to the stored list. But actually that's unnecessary as it arrives sorted, so I can remove that.